packages/gimbal/docs/ci/bitbucket/bitbucket-pipelines_gimbal-only.yml

```diff
@@ -0,0 +1,53 @@
+image: node:10.15.3
```

Comment:
What's the purpose of this file? From what I'm gathering from the readme where it says `Gimbal only bitbucket-pipelines.yml example` then this should be much smaller?

Reply:
Sorry. I pushed the wrong file.